### ISSUE TITLE
surveyjs-aspnet-mvc is not compatible with Microsoft.AspNetCore.All 2.1.5

### ISSUE THREAD

**Issue by IAmSequestered** [surveyjs/surveyjs-aspnet-mvc#1]
TL;DR: Using NuGet Manager, if you attempt to update Microsoft.AspNetCore.All beyond version 2.1.4, to version 2.1.5, then you'll get a "HTTP Error 502.5 - Process Failure" on initial page load.

After a couple of days of fighting I came up with this to reproduce the bug: 
  ** Deleted local solution and started again.
  Ran fine (Index.html loaded with list of surveys.) with first download
  Converted project to .NET Core 2.1
  - Worked fine. 
  Microsoft.AspNetCore.All update from 2.0.0 to 2.1.3
   - Worked fine. 
  Microsoft.AspNetCore.All update from 2.1.3 to 2.1.4
   - Worked fine.
  Microsoft.AspNetCore.All update from 2.1.4 to 2.1.5
   - HTTP Error 502.5 - Process Failure 
  Microsoft.AspNetCore.All downgrade from 2.1.5 to 2.1.4
   - Worked fine.

Visual Studio 2017 version 15.8.5

** Narrative:
I'm attempting to integrate Microsoft SignalR functionality into a quick patient questionnaire. After I installed Microsoft.AspNetCore.SignalR.Client 1.0.4 the project quit working. I attempted to back track on the NuGet package but that didn't fix it. Many many different attempts (2+ days worth) later I found that the project wouldn't work at all when Microsoft.AspNetCore.All was at 2.1.5 even without any other changes. I think SignalR had worked at version 2.1.3.

If I ever install any of the SignalR package and I get to the point of failure, with Microsoft.AspNetCore.All at 2.1.5 I have been unable to ever get the project to work again. 

It's my guess that there's some incompatibility with the particular version of Microsoft.AspNetCore.All. What I can't understand at all is that once NuGet does the upgrade for SignalR that it can't be undone. Seems that the NuGet manifest for SignalR is bad, in some way, when upgrading from Microsoft.AspNetCore.All 2.0.x by itself.

I'm not functional with Microsoft.AspNetCore.All at 2.1.4 and Microsoft.AspNetCore.SignalR.Client at 1.0.4

**Comment by tsv2013** [comment]
Thank you for the report. We'll take a look.

**Comment by tsv2013** [comment]
I've changed .csproj file, installed SDK 2.1.5 and it works ok for me.

**Comment by IAmSequestered** [comment]
Thanks.
